docs/en/dataset_prepare.md

```diff
@@ -376,3 +393,159 @@ python tools/convert_datasets/isaid.py /path/to/iSAID
+
+```bash
+apt-get install p7zip-full
+
+cd data_materials
+
+#extract celebAMask-HQ and split by train-set
+unzip CelebAMask-HQ.zip
+7za x CelebAMask-HQ-masks_corrected.7z -o./CelebAMask-HQ
+#suggest better code if you have
+rsync -a ./CelebAMask-HQ/CelebA-HQ-img/ --files-from=./CelebAMask-HQ-WO-train.txt ./CelebAMask-HQ-WO-Train_img
+basename -s .jpg ./CelebAMask-HQ-train/* > train.txt
+xargs -n 1 -i echo {}.png < train.txt > mask_train.txt
+rsync -a ./CelebAMask-HQ/CelebAMask-HQ-masks_corrected/ --files-from=./mask_train.txt ./CelebAMask-HQ-WO-Train_mask
+mv train.txt ../data/occlusion-aware-face-dataset
+
+#extract DTD
+tar -zxvf dtd-r1.0.1.tar.gz
+mv dtd DTD
+
+#extract hands dataset and split by 200 samples
+7za x 11k-hands_masks.7z -o.
+unzip Hands.zip
+rsync -a ./Hands/ --files-from=./11k_hands_sample.txt ./11k-hands_img
+
+#extract upscaled coco object
+7za x coco_object.7z -o.
+mv coco_object/* .
+
+#extract validation set
+7za x RealOcc.7z -o../data/occlusion-aware-face-dataset
+
+```
+
+**Dataset Organization:**
+
+```none
+
+├── data_materials
+│   ├── CelebAMask-HQ-WO-Train_img
+│   │   ├── {image}.jpg
+│   ├── CelebAMask-HQ-WO-Train_mask
+│   │   ├── {mask}.png
+│   ├── DTD
+│   │   ├── images
+│   │   │   ├── {classA}
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── {classB}
+│   │   │   │   ├── {image}.jpg
+│   ├── 11k-hands_img
+│   │   ├── {image}.jpg
+│   ├── 11k-hands_mask
+│   │   ├── {mask}.png
+│   ├── object_image_sr
+│   │   ├── {image}.jpg
+│   ├── object_mask_x4
+│   │   ├── {mask}.png
+
+├── data
+│   ├── occlusion-aware-face-dataset
+│   │   ├── train.txt
+│   │   ├── NatOcc_hand_sot
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   ├── NatOcc_object
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   ├── RandOcc
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   ├── RealOcc
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   │   ├── split
+│   │   │   │   ├── val.txt
+```
+
+## Data Generation
+
+```bash
+git clone https://github.com/jinwonkim93/face-occlusion-generation.git
+cd face_occlusion-generation
+```
+
+Example script to generate NatOcc hand dataset
+
+```bash
+CUDA_VISIBLE_DEVICES=0 NUM_WORKERS=4 python main.py \
+--config ./configs/natocc_hand.yaml \
+--opts OUTPUT_PATH "path/to/mmsegmentation/data/occlusion-aware-face-dataset/NatOcc_hand_sot"\
+AUGMENTATION.SOT True \
+SOURCE_DATASET.IMG_DIR "path/to/data_materials/CelebAMask-HQ-WO-Train_img" \
+SOURCE_DATASET.MASK_DIR "path/to/mmsegmentation/data_materials/CelebAMask-HQ-WO-Train_mask" \
+OCCLUDER_DATASET.IMG_DIR "path/to/mmsegmentation/data_materials/11k-hands_img" \
+OCCLUDER_DATASET.MASK_DIR "path/to/mmsegmentation/data_materials/11k-hands_masks"
```

Comment:
The '/' should be added to the end of the address, otherwise, the mask image will not be found.
Did you meet this problem?

Reply:
Yes, and i have fix the problem and PR it to the author. Try git pull the latest version